### ISSUE TITLE
Word Timing Accuracy Falls Off After Pauses Example

### ISSUE THREAD

**Issue by kamranjon** [linto-ai/whisper-timestamped#30]
Hey there, I love this project, thank you for all your work. I wanted to share an example of a video where the timing frequently falls behind or jumps ahead after pauses.

This is my transcribe call: `whisper.transcribe(model, audio_file, verbose=None, condition_on_previous_text=False)`
I am using the base model - the medium model has similar discrepancies but less extreme. The timestamps i provided below are from the base model.

Video: https://www.youtube.com/watch?v=jPcCG0_U6Z4

For example at 2:57 the audio says "Very unselfish pass" - whisper-timsestamped places this line at 2:54.
Another example is at 6:13 the audio says "Important flick from Davis", but whisper timestamped places this at 6:07

Hope this helps!
Thank you.

** Edit **
If you are unable to replicate any timing issues with this please feel free to close the issue

**Comment by kamranjon** [comment]
I just tested this same clip with the large-v1 model - and the discrepancies are really not bad at all - I am going to close this as it seems to be related to whisper more than whisper-timestamped

**Comment by kamranjon** [comment]
Here is an example pulled out of that video:

If you run this through whisper-timestamped - at 3:19 it skips the words `number ten` and the rest of the transcript is off - I wonder if this could be related to some of the offsets.

https://user-images.githubusercontent.com/3966239/218189493-2edc3bc0-0822-4719-836f-41ad134b2148.mp4

**Comment by Jeronymous** [comment]
Thank you @kamranjon for reporting this and giving the audio to reproduce (BTW, I'm not able to access the Youtube video in my region).
Unfortunately, I'm not able to reproduce what you describe.

If you want to double check, here is what I obtained, with large model v1, respectively with condition_on_previous_text=False and True [issue_30.zip](https://github.com/linto-ai/whisper-timestamped/files/10722588/issue_30.zip).
I launched:
```
python3 <...>/whisper_timestamped/transcribe.py <...>.mp4 --model large-v1 --condition_on_previous_text True --output_dir issue_30/large-v1.True --output_format json
python3 <...>/whisper_timestamped/transcribe.py <...>.mp4 --model large-v1 --condition_on_previous_text False --output_dir issue_30/large-v1.False --output_format json
```
(and I work with version 1.7.9 of whisper_timestamped)

I also tried models base and large-v2. Got no chance to reproduce the problem you had at that particular moment around "number 10" at 3:19
